doc/source/visualizing/plots.rst

```diff
@@ -30,7 +30,7 @@ example in a jupyter notebook environment, but the same command should work
 in other environments as well:
```

Comment:
Github won't let me comment on the line I want you to edit directly, but up at the top of this page you should mention that we support 1D line plots along with the other plots.